internal/tweets/tweetsStr.go

```diff
@@ -0,0 +1,36 @@
+package tweets
+
+import (
+	"github.com/go-playground/validator/v10"
+	"time"
+)
+
+type Tweet struct {
+	TweetID             int       `json:"tweet_id"`
+	UserID              int       `json:"user_id"`
+	Author              string    `json:"author"`
+	Text                string    `json:"text"`
+	CreatedAt           time.Time `json:"created_at"`
+	LikeCount           int       `json:"like_count"`
+	Retweet             int       `json:"repost"`
+	Public              bool      `json:"public"`
+	OnlyFollowers       bool      `json:"only_followers"`
+	OnlyMutualFollowers bool      `json:"only_mutual_followers"`
+	OnlyMe              bool      `json:"only_me"`
+	LoginToken          string
+	ParentTweetId       int `json:"parent_tweet_id"`
+}
+type CreatNewTweet struct {
+	TweetID             int
+	Text                string `json:"text" validate:"omitempty,checkTweetText"`
```

Comment:
that would mean that empty text is ok, which I think is not

Reply:
done